golem_messages/message/tasks.py

```diff
@@ -330,33 +329,11 @@ def subtask_id(self):
     def task_to_compute(self):
         return self
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        if not hasattr(self, '_ethsig'):
-            self._ethsig = None
-
-    def serialize(self, *args, **kwargs):  # pylint: disable=arguments-differ
-        serialized = super().serialize(*args, **kwargs)
-        sig_length = self.ETHSIG_LENGTH - 1
```

Comment:
`ETHSIG_LENGTH` & `ETHSIG_FORMAT` are not needed anymore (remove from class)

Reply:
:+1: done